cypress/e2e/news/createReadDeleteNewsInAdvance.feature

```diff
@@ -0,0 +1,49 @@
+@stable_test
+@regression_test
+Feature:  News - To read a news on the respective dashboards
+
+    As a teacher I want to read the news shown on the dashboard so that I'm informed about the latest news
+
+    Scenario: User creates news with time options, reads them and deletes them
+        # as a pre-condition create users for author and reader
```

Comment:
_pre-condition: create users for teacher and student_

Reply:
This test would be possible for different roles, so I want to keep it flexibel and not to define a special role.